wiki/Main_Page/de.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: main_page
+---
+
+<!-- Do not add any empty lines inside this div. -->
+
+<div class="wiki-main-page__blurb">
+Willkommen zu der osu! wiki. Ein Projekt was eine große Reichweite an osu! basierten Informationen beinhaltet.
+</div>
+
+<div class="wiki-main-page__panels">
+<div class="wiki-main-page-panel wiki-main-page-panel--full">
+
+# Erste Schritte
+
+[Willkommen](/wiki/Welcome) • [Installation](/wiki/Installation) • [Registrierung](/wiki/Registration) • [Hilfe-Center](/wiki/Help_Centre) • [FAQ](/wiki/FAQ)
+
+</div>
+<div class="wiki-main-page-panel">
+
+# Spiel-Client
+
+[Benutzeroberfläche](/wiki/Interface) • [Optionen](/wiki/Options) • [Visuelle Einstellungen](/wiki/Visual_Settings) • [Tastenkürzel-Referenzen](/wiki/Shortcut_key_reference) • [Konfigurationsdatei](/wiki/osu!_Program_Files/User_Configuration_File) • [Programmdateien](/wiki/osu!_Program_Files)
+
+[Dateiformate](/wiki/osu!_File_Formats): [.osz](/wiki/osu!_File_Formats/Osz_(file_format)) • [.osk](/wiki/osu!_File_Formats/Osk_(file_format)) • [.osr](/wiki/osu!_File_Formats/Osr_(file_format)) • [.osu](/wiki/osu!_File_Formats/Osu_(file_format)) • [.osb](/wiki/osu!_File_Formats/Osb_(file_format)) • [.db](/wiki/osu!_File_Formats/Db_(file_format))
+
+</div>
+<div class="wiki-main-page-panel">
+
+# Spielverlauf
+
+[Spielmodis](/wiki/Game_mode): [osu!](/wiki/Game_mode/osu!) • [osu!taiko](/wiki/Game_mode/osu!taiko) • [osu!catch](/wiki/Game_mode/osu!catch) • [osu!mania](/wiki/Game_mode/osu!mania)
+
+[Beatmap](/wiki/Beatmap) • [Hit-Objekte](/wiki/Hit_object) • [Mods](/wiki/Game_modifier) • [Punktzahl](/wiki/Score) • [Wiederholung](/wiki/Replay) • [Multi](/wiki/Multi)
+
+</div>
+<div class="wiki-main-page-panel">
+
+# [Beatmap editor](/wiki/Beatmap_Editor)
+
+Sections: [Zusammenstellung](/wiki/Beatmap_Editor/Compose) • [Design](/wiki/Beatmap_Editor/Design) • [Timing](/wiki/Beatmap_Editor/Timing) • [Song-Einstellung](/wiki/Beatmap_Editor/Song_Setup)
+
+Komponenten: [AiMod](/wiki/Beatmap_Editor/AiMod) • [Schlagschnellteiler](/wiki/Beatmap_Editor/Beat_Snap_Divisor) • [Schlagschnellteiler](/wiki/Beatmap_Editor/Distance_Snap) • [Menu](/wiki/Beatmap_Editor/Menu) • [SB Ladung](/wiki/Beatmap_Editor/SB_Load) • [Zeitleisten](/wiki/Beatmap_Editor/Timelines)
+
+[Beatmapping](/wiki/Beatmapping) • [Schwierigkeitsgrade](/wiki/Beatmap/Difficulty) • [Mapping Techniken](/wiki/Mapping_Techniques) • [Storyboarding](/wiki/Storyboarding)
+
+</div>
+<div class="wiki-main-page-panel">
+
+# Beatmap-Einreichung und Ranking
+
+[Einreichen](/wiki/Submission) • [Modding](/wiki/Modding) • [Ranking-Verfahren](/wiki/Beatmap_ranking_procedure) • [Mappers' Gilde](/wiki/Mappers_Guild) • [Project Loved](/wiki/Project_Loved)
```

Comment:
I guess you could either look at it as a name and don't translate it (maybe preferred here)
```suggestion
[Einreichen](/wiki/Submission) • [Modding](/wiki/Modding) • [Ranking-Verfahren](/wiki/Beatmap_ranking_procedure) • [Mappers' Guild](/wiki/Mappers_Guild) • [Project Loved](/wiki/Project_Loved)
```
or translate everything
```suggestion
[Einreichen](/wiki/Submission) • [Modding](/wiki/Modding) • [Ranking-Verfahren](/wiki/Beatmap_ranking_procedure) • [Mapper-Gilde](/wiki/Mappers_Guild) • [Project Loved](/wiki/Project_Loved)
```